rca/schools/models.py

```diff
@@ -509,9 +510,23 @@ def get_related_programmes(self):
 
     def get_programme_index_link(self):
         ProgrammeIndexPage = apps.get_model("programmes", "ProgrammeIndexPage")
-        programm_index = ProgrammeIndexPage.objects.live().first()
-        if programm_index:
-            return f"{programm_index.get_url()}?category=related_schools_and_research_pages&value={self.pk}-{self.slug}"
+        programme_index = ProgrammeIndexPage.objects.live().first()
+        return programme_index.get_url()
+
+    def get_short_courses_index_link(self):
+        """Returns a link to the programme index page filtered by the
+        Short Course ProgrammeType
+        """
+        ProgrammeType = apps.get_model("programmes", "ProgrammeType")
+        short_course_type = ProgrammeType.objects.filter(
+            display_name="Short course"
+        ).first()
+        if not short_course_type:
+            return
+        return (
+            f"{self.get_programme_index_link()}?category=programme_type&"
+            f"value={str(short_course_type.id)}-{slugify(short_course_type.display_name)}"
```

Comment:
Does `str()` need to be called here?

Reply:
Nope - hangover from when I didn't have this as an `f` string